### DISCUSSION TITLE
Pyodide V 19 Scipy issue

### DISCUSSION THREAD

**Discussion by tjefferson401** [pyodide/pyodide#3209] [Q&A]
Hi, please let me know if this is the wrong location to ask this question, which goes as follows:

I'm building pyodide on local, and using pyodide version 0.19.X. I'm building individual packages as I need them, and built the Scipy package successfully. The issue comes when loading the package, a string of warnings are thrown:

`
Couldn't instantiate wasm: /lib/python3.9/site-packages/scipy/linalg/_flapack.so 'RuntimeError: abort(Assertion failed: bad export type for srotg_: undefined). Build with -s ASSERTIONS=1 for more info.'
(anonymous) @ pyodide.asm.js:14
Promise.then (async)
handle @ pyodide.asm.js:14
(anonymous) @ pyodide.asm.js:14
(anonymous) @ pyodide.asm.js:14
loadPackage @ pyodide.asm.js:14
processPackageData @ scipy.js:1
(anonymous) @ scipy.js:1
xhr.onload @ scipy.js:1
load (async)
fetchRemotePackage @ scipy.js:1
loadPackage @ scipy.js:1
(anonymous) @ scipy.js:1
(anonymous) @ scipy.js:1
`
`
pyodide.asm.js:14 Couldn't instantiate wasm: /lib/python3.9/site-packages/scipy/linalg/_flinalg.so 'RuntimeError: abort(Assertion failed: bad export type for srotg_: undefined). Build with -s ASSERTIONS=1 for more info.'
(anonymous) @ pyodide.asm.js:14
Promise.then (async)
handle @ pyodide.asm.js:14
(anonymous) @ pyodide.asm.js:14
(anonymous) @ pyodide.asm.js:14
loadPackage @ pyodide.asm.js:14
processPackageData @ scipy.js:1
(anonymous) @ scipy.js:1
xhr.onload @ scipy.js:1
load (async)
fetchRemotePackage @ scipy.js:1
loadPackage @ scipy.js:1
(anonymous) @ scipy.js:1
(anonymous) @ scipy.js:1
`
`
pyodide.asm.js:14 Couldn't instantiate wasm: /lib/python3.9/site-packages/scipy/linalg/_interpolative.so 'RuntimeError: abort(Assertion failed: bad export type for srotg_: undefined). Build with -s ASSERTIONS=1 for more info.'
...
`
Several more warnings in identical format appear.

The code I am running that yields the error looks pretty much like this:

```
async function setPyodide() {  
  window.pyodide = await loadPyodide({  
    indexURL: PYODIDE_BUILD,  
  });  
  await self.pyodide.loadPackage(`${PYODIDE_BUILD}numpy.js`);  
  await self.pyodide.loadPackage(`${PYODIDE_BUILD}scipy.js`);  
  return "Pyodide Initialization Complete";  
}
```
scipy.js is found at the path coded as well (${PYODIDE_BUILD}scipy.js).

The closest issue I've found is #1657 which shows a similar issue with pygame, however I am having trouble interpreting the solution stated there. Has anyone seen this before or may be able to direct me to the location of the issue?

### Environment Specs
Pyodide Version: 19.X
OS: Mac OS 12.3.1
Chrome: Version 107.0.5304.62 (Official Build) (x86_64)

**Comment by ryanking13** [comment]
Could you try with the latest version of Pyodide? The Pyodide codebase has changed a lot since v0.19 so it is probably fixed in newer versions.

**Reply by tjefferson401** [reply]
Hi, thanks for the response! Another package I am using is written for 3.9 hence my need for Pyodide v0.19.

**Reply by ryanking13** [reply]
Oh, okay. That makes sense. However, we don't backport bug patches into older versions, so if this is a bug in v0.19, I'm sorry but you'll have to use newer versions of Pyodide.

**Reply by tjefferson401** [reply]
Understood, thank you for the response!
